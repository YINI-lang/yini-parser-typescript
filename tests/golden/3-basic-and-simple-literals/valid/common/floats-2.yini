^ Numbers
float4  = 1e10               // 10,000,000,000 (exponential)
float5  = -1.234e-12         // Small negative, scientific notation
float6  = 123456789.987654321 // Large float, many digits
float7  = 1.7976931348623157e+308   // Max JS/IEEE 754 double (Number.MAX_VALUE)
float8  = 5e-324             // Min positive subnormal JS/IEEE 754 double (Number.MIN_VALUE)
