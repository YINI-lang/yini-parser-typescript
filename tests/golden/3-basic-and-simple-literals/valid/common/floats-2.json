{
    "Numbers": {
        "float4": 10000000000,
        "float5": -1.234e-12,
        "float6": 123456789.98765433,
        "float7": 1.7976931348623157e+308,
        "float8": 5e-324
    }
}
